lib/__tests__/topic.spec.js

```diff
@@ -4,42 +4,54 @@ import { parseTopic } from '../topic';
 
 vi.mock('sharp');
 
-describe('topic', () => {
-  it('should reject when dir not lowercase', () => {
-    const p = resolveFixturePath('a-TOPIC');
-    expect.assertions(1);
-    return parseTopic(p).catch((err) => {
-      expect(err.message)
-        .toBe('Directory name must be all lowercase and received a-TOPIC');
+describe('parseTopic', () => {
+  it('should add the specified language keys to intl', () => {
+    const p = resolveFixturePath('topic-with-translations');
+    expect.assertions(3);
+    return parseTopic(p, {}, {}).then((topic) => {
+      expect(Object.keys(topic.intl).length).toBe(2);
+      expect(topic.intl).toHaveProperty('es');
+      expect(topic.intl).toHaveProperty('pt');
     });
   });
 
-  it('should reject when dir contains invalid chars', () => {
-    const p = resolveFixturePath('a%-topic');
-    expect.assertions(1);
-    return parseTopic(p).catch((err) => {
-      expect(err.message)
-        .toBe('Directory name must only contain alphanumeric characters and hyphens (-)');
+  describe('should reject', () => {
+    it('when dir not lowercase', () => {
```

Comment:
solo agrupe los rejects en `should reject` aqui. puedo volver al anterior si genera mucho ruido

Reply:
No me parece mala idea agruparlos... pero efectivamente queda un poco raro junto con los otros tests...

![image](https://user-images.githubusercontent.com/110297/214250980-fdb19628-ff94-4b0b-ba23-890d77990b42.png)

Ahora que el PR de @mfdebian está integrado, me he tomado la libertad de reorganizar esos tests también, quedando algo como :point_down: 

![image](https://user-images.githubusercontent.com/110297/214251502-ff6081d7-2ff0-45dd-8e24-db6610e48639.png)